training/src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,25 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST, reason = "Actor Not Found")
+public class BookIdMismatchException extends RuntimeException {
+
+    public BookIdMismatchException() {
+
+        super();
+    }
+    public BookIdMismatchException(String message, Throwable cause) {
+
+        super(message, cause);
+    }
+    public BookIdMismatchException(String message) {
+
+        super(message);
+    }
+    public BookIdMismatchException(Throwable cause) {
+
+        super(cause);
```

Comment:
You can declare the message in the constructor
```suggestion
        super("The book's id does not correspond to the data to be updated");
```